addons/true-youtube-links/addon.json

```diff
@@ -9,7 +9,7 @@
   "userscripts": [
     {
       "url": "userscript.js",
-      "matches": ["projects", "studios", "profiles", "forums"],
+      "matches": ["https://scratch.mit.edu/discuss/*"],
```

Comment:
```suggestion
      "matches": ["projects", "studios", "profiles", "https://scratch.mit.edu/discuss/*"],
```

Reply:
Sure but the extra matches seem unnessaey since more links doesn't change any URLs.

Reply:
What?

Reply:
When would true YouTube links need to run outside the forums? Maybe video links in Scratch news but the addon doesn't run on the home page.

Reply:
Some people put links to the forum YouTube player into project and studio descriptions because Scratch makes them clickable. Example: https://scratch.mit.edu/projects/574340251/